crates/stages/src/stages/headers.rs

```diff
@@ -120,7 +120,7 @@ where
         // Although headers were downloaded in reverse order, the collector iterates it in ascending
         // order
 
-        let interval = total_headers / 10;
+        let interval = (total_headers / 10).max(1);
```

Comment:
where's the division by zero here?

Reply:
rem panics
https://github.com/paradigmxyz/reth/blob/c552897e9f5c4a83bfb461ff6f29485c0db4771b/crates/stages/src/stages/headers.rs#L127

Reply:
when does this happen?

if `total_headers` is 0 then this will still be 0?

this opens up another in this if check because anything % 1 is 0

Reply:
if `total_headers < 10`

Reply:
> this opens up another in this if check because anything % 1 is 0

which is fine because we'll report the progress on every iteration